releng/com.b2international.snowowl.server.update/assembly/common/bin/dmk.sh

```diff
@@ -186,52 +175,40 @@ then
 	then
 		TMP_DIR=$KERNEL_HOME/work/tmp
 		# Ensure that the tmp directory exists
-		mkdir -p $TMP_DIR
+		mkdir -p "$TMP_DIR"
 		
-		#Added awt.headless - http://mail-archives.apache.org/mod_mbox/poi-user/200705.mbox/%3C15719338671.20070504144714@dinom.ru%3E
-        JAVA_OPTS="$JAVA_OPTS \
```

Comment:
This was our custom default JVM configuration, please bring it back. Also check the other files as well, again!